inference-engine/tests/functional/plugin/cpu/single_layer_tests/crop.cpp

```diff
@@ -0,0 +1,177 @@
+        result << "end=" << CommonTestUtils::vec2str(params.end) << "_";
+        result << "stride=" << CommonTestUtils::vec2str(params.strides) << "_";
+        result << "begin_m=" << CommonTestUtils::vec2str(params.beginMask) << "_";
+        result << "end_m=" << CommonTestUtils::vec2str(params.endMask) << "_";
+        if (!params.newAxisMask.empty()) {
+            result << "new_axis_m=" << (params.newAxisMask.empty() ? "def" : CommonTestUtils::vec2str(params.newAxisMask)) << "_";
+        }
+        if (!params.shrinkAxisMask.empty()) {
+            result << "shrink_m=" << (params.shrinkAxisMask.empty() ? "def" : CommonTestUtils::vec2str(params.shrinkAxisMask)) << "_";
+        }
+        if (!params.ellipsisAxisMask.empty()) {
+            result << "ellipsis_m=" << (params.ellipsisAxisMask.empty() ? "def" : CommonTestUtils::vec2str(params.ellipsisAxisMask)) << "_";
+        }
+        result << "trgDev=" << targetName;
+        result << CPUTestsBase::getTestCaseName(cpuParams);
+
+        return result.str();
+    }
+protected:
+    void SetUp() override {
+        StridedSliceSpecificParams ssParams;
+        InferenceEngine::Precision netPrecision;
+        std::map<std::string, std::string> additionalConfig;
+        CPUSpecificParams cpuParams;
+        std::tie(ssParams, netPrecision, targetDevice, additionalConfig, cpuParams) = this->GetParam();
+        inPrc = outPrc = netPrecision; // because crop does not convert Precisions, but only moves the data
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+        configuration.insert(additionalConfig.begin(), additionalConfig.end());
+
+        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+        auto params = ngraph::builder::makeParams(ngPrc, {ssParams.inputShape});
+        auto paramOuts = ngraph::helpers::convert2OutputVector(
+                ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
+        auto ss = ngraph::builder::makeStridedSlice(paramOuts[0], ssParams.begin, ssParams.end, ssParams.strides, ngPrc, ssParams.beginMask,
+                                                    ssParams.endMask, ssParams.newAxisMask, ssParams.shrinkAxisMask, ssParams.ellipsisAxisMask);
+
+        selectedType = std::string("unknown_") + inPrc.name();
+
+        ss->get_rt_info() = getCPUInfo();
+
+        ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(ss)};
+        function = std::make_shared<ngraph::Function>(results, params, "StridedSlice");
+    }
+};
+
+TEST_P(CropLayerCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    Run();
+    CheckCPUImpl(executableNetwork, "Crop");
+}
+
+namespace {
+// Within the test scope we don't need any implicit bf16 optimisations, so let's run the network as is.
+const std::map<std::string, std::string> additional_config = {{PluginConfigParams::KEY_ENFORCE_BF16, PluginConfigParams::NO}};
```

Comment:
1. I agree that code repetition is bad. The common plugin configuration has been moved to the more appropriate place in 69ca3b4015fd4b932c377d389c57063544f17088.
2. In my opinion, we should disable implicit BF16 optimizations in single layer tests. There is two reasons. On the one hand, we are testing the layer itself, and the less additional procedures are added to the execution flow, the better. Moreover, the behavior of bf16transformer can be changed over time or it can be removed at all. Therefore, we do not want to depend on such a subsystem in single-layer tests. On the other hand, when we run FP32 single layer tests on a platform that supports bfloat16 with bf16 optimizations enabled, this test may be run with bf16 precision implicitly for us, which is undesirable behavior. So we have to disable bf16 optimizations to run single layer tests with the precision specified in the tests definitions and not any other.